images/dvcr-artifact/pkg/uploader/uploader_fuzz_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2025 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package uploader
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/deckhouse/virtualization-controller/dvcr-importers/pkg/fuzz"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	cryptowatch "kubevirt.io/containerized-data-importer/pkg/util/tls-crypto-watch"
+)
+
+const (
+	DEFAULT_PORT = 8000
+	MOCK_PORT    = 8400
```

Comment:
As I understand, actual values for these ports are not important. Can we make them "random"? You never know if hardcoded ports are used in CI/CD or local environment. Open two listener sockets with no port and use them with http.Serve.
Something like this:
```
listener, err := net.Listen("tcp", "localhost:0")
...
http.Serve(listener, mux)
```

Connection string for the client is available in listener.Addr().String()